test/session.js

```diff
@@ -897,9 +897,15 @@ describe('session()', function(){
   })
 
   describe('rolling option', function(){
-    it('should default to false', function(done){
+    it('should default to false and not roll cookie', function(done){
```

Comment:
This should be it's own test, rather than altering an existing test. I can always help if you're not sure how to make one and that's why an existing test was altered vs a new one.